docu/examples/example8-uncaught-runtime-errors.ipf

```diff
@@ -6,17 +6,23 @@
 
 Function TestWaveOp()
 
-		WAVE/Z/SDFR=$"I dont exist" wv;
+		WAVE/Z/SDFR=$"I dont exist" wv
+		print "This will be printed, even if a RTE occurs."
+
+		WAVE/Z/SDFR=$"I dont exist" wv; AbortOnRTE
```

Comment:
If you want to show that the previous RTE is overwritten it would be good to have a different RTE condition before and having the related RTE error codes mentioned in the comment/description.